tests/telemetry/test_writer.py

```diff
@@ -37,7 +37,8 @@ def test_add_event(telemetry_writer, test_agent_session, mock_time):
     assert requests[0]["headers"]["DD-Telemetry-Request-Type"] == payload_type
     assert requests[0]["headers"]["DD-Telemetry-API-Version"] == "v2"
     assert requests[0]["headers"]["DD-Telemetry-Debug-Enabled"] == "False"
-    assert requests[0]["body"] == _get_request_body(payload, payload_type)
+    seq_id = requests[0]["body"]["seq_id"]
```

Comment:
NB: What is this exactly?

Reply:
seq_id is the sequence id. It signifies the order of telemetry requests. The first request sent will have a sequence id of 1, the next will be 2, .....

This field allows the backend to track the order events were sent.